ScoutSuite/output/data/html/partials/gcp/left_menu_for_gcp_region.html

```diff
@@ -0,0 +1,38 @@
+<!-- Generic left-menu link partial for region-then-resource -->
+<script id="left_menu_for_gcp_region.partial" type="text/x-handlebars-template">
+  <div id="services.{{service_name}}.projects.id.regions.id.{{resource_type}}.list">
+  <div class="list-group" >
+    <div class="list-group-item">
+      <a href="javascript:showRowWithItems('services.{{service_name}}.projects.id.regions.id.{{resource_type}}')">Show all <span class="badge float-right btn-info">{{get_value_at 'metadata' service_group service_name 'resources' resource_type 'count'}}</span></a>
```

Comment:
`get_value_at` was renamed, it's now `getValueAt` I believe.

Reply:
Merged and took care of it!